IFC4x3/Properties/i/InjectionPressure_29eahe7Wr9cw6L65Rn8pJw/Documentation.md

```diff
@@ -0,0 +1 @@
+Injection pressure.
```

Comment:
These kinds of descriptions are quite lackluster. How about "The required pressure for injecting drillholes."

Shooting from my hip here, but please consider making them better.